hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleanActionExecutor.java

```diff
@@ -230,11 +230,15 @@ public HoodieCleanMetadata execute() {
         .filterInflightsAndRequested().getInstants().collect(Collectors.toList());
     if (pendingCleanInstants.size() > 0) {
       pendingCleanInstants.forEach(hoodieInstant -> {
-        LOG.info("Finishing previously unfinished cleaner instant=" + hoodieInstant);
-        try {
-          cleanMetadataList.add(runPendingClean(table, hoodieInstant));
-        } catch (Exception e) {
-          LOG.warn("Failed to perform previous clean operation, instant: " + hoodieInstant, e);
+        if (table.getCleanTimeline().isEmpty(hoodieInstant)) {
```

Comment:
if we have taken care of empty instants during planning phase, we should not see empty instants during execution phase right?

Reply:
I think it should be, but at present, we do encounter the problem of empty files in many cases, as well as in the test cases of PR run I submitted earlier

Reply:
not to self: here for every clean, either inflight or requested will be returned (if not completed) depending on its current state. so, either we will delete the inflight instant only or delete the requested instant.

Reply:
what happens if inflight is empty, but requested is not. I understand we will delete the inflight here. but will the instant get re-attempted to be clean later (since the requested meta file is intact) ?